packages/code-du-travail-data/dataset/fiches_ministere_travail/report.js

```diff
@@ -1,6 +1,6 @@
 const fiches = require("./fiches-min-travail.json");
 
-const hasNoTheme = fiche => !fiche.themeCdtn;
+const hasNoTheme = fiche => !fiche.parents || fiche.parents.length === 0;
```

Comment:
I would prefer a themes key, to me, parent implies a "fiche" could have children

Reply:
what about `breadcrumbs` ?